src/entity2input.js

```diff
@@ -1,19 +1,21 @@
 const { camelCase, upperFirst } = require('lodash')
-const { entityField2gql } = require("./helpers/gqlConverters")
+const { entityField2gql, EnumConventions } = require("./helpers/gqlConverters")
 const { checker } = require('suma')
 const { entityValidator } = require('./herbsValidator')
 
-function entity2input(entity, isCamelCase = true, customName ) {
+function entity2input(entity, options = undefined) {
   const validation = entityValidator(entity)
   if (!checker.isEmpty(validation)) {
     const error = Error('InvalidEntity')
     error.invalidArgs = validation
     throw error
   }
 
+  const convention  = EnumConventions[options?.convention]
```

Comment:
it should be a merge of the current/default convention to the inject convention: 
something like `const convention = Object.assign({}, defaultConvention, options?.convention))`

Reply:
solved this way
`const convention = options?.convention || defaultConvention`